webapp/javascript/components/TimelineChartWrapper.tsx

```diff
@@ -140,7 +143,7 @@ class TimelineChartWrapper extends React.Component<
       const to = new Date(formatAsOBject(m.to)).getTime();
 
       // We make the sides thicker to indicate the boundary
-      const boundary = { lineWidth: 3, color: m.color.rgb() };
+      const boundary = { lineWidth: 1, color: m.color.rgb() };
```

Comment:
Question: is this still required? The whole point was to have a "thicker" line. Now if they are exactly the same as normally, then it may not be needed no?